concat-css.php

```diff
@@ -112,11 +128,11 @@ function do_items( $handles = false, $group = false ) {
 					$media = 'all';
 				}
 
-				if ( ! isset( $stylesheets[ $stylesheet_group_index ] ) || ( isset( $stylesheets[ $stylesheet_group_index ] ) && ! is_array( $stylesheets[ $stylesheet_group_index ] ) ) ) {
-					$stylesheets[ $stylesheet_group_index ] = array();
+				if ( ! isset( $stylesheets[ $concat_group ] ) || ( isset( $stylesheets[ $concat_group ] ) && ! is_array( $stylesheets[ $concat_group ] ) ) ) {
+					$stylesheets[ $concat_group ] = array();
```

Comment:
Why is this necessary? We explicitly initialize `$stylesheets[ $concat_group ]` [here](https://github.com/Automattic/page-optimize/pull/10/files?w=1#diff-1ffea94758f96baec2cef05252ccf839R41).

Reply:
Yeah, left it from going back and forth with testing. Totally not needed.